Source/Shaders/BillboardCollectionVS.glsl

```diff
@@ -71,23 +71,23 @@ vec4 computePositionWindowCoordinates(vec4 positionEC, vec2 imageSize, float sca
         positionEC.xy += halfSize;
     }
 
-    vec4 positionWC = czm_eyeToWindowCoordinates(positionEC);
+    float mpp = czm_metersPerPixel(positionEC);
```

Comment:
Is it possible to only compute this if `sizeInMeters` is true and still function properly with the new changes? If not, not a big deal.

Reply:
No, since it's needed regardless of whether `sizeInMeters` is true or false;